backend-checks/src/main/scala/uk/gov/nationalarchives/tdr/localaws/backendchecks/checks/AntivirusCheck.scala

```diff
@@ -1,62 +1,26 @@
 package uk.gov.nationalarchives.tdr.localaws.backendchecks.checks
 
-import java.nio.file.{Path, Paths}
-import java.time.Instant
 import java.util.UUID
 
-import graphql.codegen.AddAntivirusMetadata.AddAntivirusMetadata
-import graphql.codegen.GetOriginalPath.getOriginalPath
-import graphql.codegen.types.AddAntivirusMetadataInput
-import uk.gov.nationalarchives.tdr.GraphQLClient
+import uk.gov.nationalarchives.tdr.localaws.backendchecks.api.FileService
 import uk.gov.nationalarchives.tdr.localaws.backendchecks.auth.TokenService
+import uk.gov.nationalarchives.tdr.localaws.backendchecks.checks.metadata.FakeAntivirusMetadata
 
 import scala.concurrent.{ExecutionContext, Future}
 
 class AntivirusCheck(
-                        tokenService: TokenService,
```

Comment:
I've refactored this class to make the process clearer, and to share some of the logic (like looking up the original file path) with the checksum check.